### DISCUSSION TITLE
How to receive real-time video processed with OpenCV

### DISCUSSION THREAD

**Discussion by ikekou0805** [AxisCommunications/acap-native-sdk-examples#281] [General]
Currently, I am looking to receive data processed with OpenCV on the client PC side by using the acap-native-sdk-examples using-opencv code. I am using the M2036 camera (CPU: aarch64), and I have confirmed that ACAP and OpenCV are both operational. 

However, I was unable to find functions in the vdo library to pass the preprocessed data to the transmission stream. As a workaround, I tried sending the camera frames one by one using socket communication, but the video frame rate is low, and I cannot receive it in real-time. 

If you are aware of any effective functions or methods, I would appreciate your guidance.

**Comment by pataxis** [comment]
Hi @ikekou0805 , thanks for reaching out!

I would refer to the [Video Capture API](https://axiscommunications.github.io/acap-documentation/docs/api/native-sdk-api.html#video-capture-api-vdo) documentation but it sounds like you have already looked at that. You could try to ask in the AxisCommunications [Discussions](https://github.com/orgs/AxisCommunications/discussions) which is more active, someone might have tested out what you're trying to achieve.
